src/recipe.hpp

```diff
@@ -62,6 +62,8 @@ class recipe {
 
     virtual std::vector<cell_connection> connections_on(cell_gid_type) const = 0;
     virtual probe_info get_probe(cell_member_type probe_id) const = 0;
+
+    virtual util::any get_global_properties(cell_kind) const { return util::any{}; };
```

Comment:
Because this returns an `any`, a comment explaining what is expected inside the `any` is needed.

Reply:
Like the cell descriptions, it will change from cell kind to cell kind. But I can add a comment to that effect.